epub33/rs/index.html

```diff
@@ -2697,6 +2697,11 @@ <h3>Substantive changes since <a href="https://www.w3.org/TR/2022/CR-epub-rs-33-
 						Recommendation</a></h3>
 
 				<ul>
+					<li>11-Jan-2023: Added note advising reading system developers to support package metadata
+						directionality even though marked as under-implemented for authoring. See <a
+							href="https://github.com/w3c/epub-specs/pull/2515">pull request 2515</a>.</li>
+					<li>11-Jan-2023: Removed support description for under-implemented features. See <a
+							href="https://github.com/w3c/epub-specs/pull/2515">pull request 2515</a>.</li>
```

Comment:
I guess this is based on getting to an equilibrium point on MO, ie, that there is no under-implemented notion for MO at all (which is probably all right, because MO is not a required feature as a whole).

Reply:
Either that or we do something similar where we under-implement the authoring feature. I made that change after the last call when it sounded like we wouldn't have to label MO because it's only a recommended feature. It's easy to get back the prose if we need it, though.